docs/manuals/source/TasksAndConcepts/MigrationAndCopy.rst

```diff
@@ -39,7 +39,7 @@ A migration job can be started manually or from a Schedule, like a backup job. I
 
 Normally three jobs are involved during a migration:
 
--  The Migration control Job which starts the migration child Jobs.
+-  The Migration control Job which starts the migration child Jobs (removed upon completion).
```

Comment:
Is the job also removed if completes with error?

Reply:
Also, do we want to add the information that the "migration child jobs" will be removed when the original jobid is removed from the database?

Reply:
when the job enters `MigrationCleanup(...)` in migrate.cc:1659 at the end of the job, it will be deleted.
A migration cleanup is part of the natural cycle of the job, so whether successful or failed, it gets called, and thus the control job gets removed.

Reply:
> Also, do we want to add the information that the "migration child jobs" will be removed when the original jobid is removed from the database?

I'll see where I can fit that information in the docs